lib/ohai/plugins/windows/network.rb

```diff
@@ -53,6 +53,42 @@ def msft_adapter_data
     data
   end
 
+  # Returns the interface code in Hexadecimal Format
```

Comment:
YARD format would be helpful here in the future

Reply:
Updating them
Thank you